modules/recommended-configurable-storage-technology.adoc

```diff
@@ -84,39 +84,45 @@ In a non-scaled/high-availability (HA) {product-title} registry cluster deployme
 
 * The preferred storage technology is object storage followed by block storage. The
 storage technology does not have to support RWX access mode.
-* The storage technology must ensure read-after-write consistency. All NAS storage are not
-recommended for {product-title} Registry cluster deployment with production workloads.
-* While `hostPath` volumes are configurable for a non-scaled/HA {product-title} Registry, they are not recommended for cluster deployment.
+* The storage technology must ensure read-after-write consistency. File storage is
+not recommended for {product-title} Registry cluster deployment with production
+workloads.
+* While `hostPath` volumes are configurable for a non-scaled/HA {product-title}
+Registry, they are not recommended for cluster deployment.
 
 === Scaled registry
 
 In a scaled/HA {product-title} registry cluster deployment:
 
-* The preferred storage technology is object storage. The storage technology must support RWX access mode and must ensure read-after-write consistency.
-* File storage and block storage are not recommended for a scaled/HA {product-title} registry cluster deployment with production workloads.
+* The preferred storage technology is object storage.
+* Object storage must be S3 compliant.
+* The storage technology must support RWX access mode and must ensure read-after-write consistency.
+* File storage is not recommended for a scaled/HA {product-title} registry cluster
+deployment with production workloads.
+* Block storage is not configurable.
 
 === Metrics
 
 In an {product-title} hosted metrics cluster deployment:
 
 * The preferred storage technology is block storage.
+* Object storage is not configurable.
+
 
 [IMPORTANT]
 ====
-Testing shows significant unrecoverable corruptions using file storage and,
-therefore, file storage is not recommended for use with metrics.
-
-There are file storage implementations in the marketplace that might not have
-these issues. Contact the individual storage vendor for more information on any
-testing that was possibly completed against these OpenShift core components.
+It is not recommended to use file storage for a hosted metrics cluster
+deployment with production workloads.
 ====
 
 === Logging
 
 In an {product-title} hosted logging cluster deployment:
 
 * The preferred storage technology is block storage.
-* It is not recommended to use NAS storage for a hosted metrics cluster deployment with production workloads.
+* It is not recommended to use NAS storage for a hosted metrics cluster deployment
```

Comment:
Should it say "File storage is recommended [...]" instead, like in Registry above? NAS can provide block storage too.

Reply:
There is a typo in this paragraph about Logging.
Sentence should be:
File storage is not recommended for a hosted **logging** cluster... 
(Not metrics)